x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/outlier_hyper_parameters.tsx

```diff
@@ -30,7 +30,7 @@ export const OutlierHyperParameters: FC<Props> = ({ actions, state, advancedPara
           })}
           helpText={i18n.translate('xpack.ml.dataframe.analytics.create.methodHelpText', {
             defaultMessage:
-              'Sets the method that outlier detection uses. If not set, uses an ensemble of different methods and normalises and combines their individual outlier scores to obtain the overall outlier score. We recommend to use the ensemble method',
+              'Sets the method that outlier detection uses. If not set, uses an ensemble of different methods, normalises and combines their individual outlier scores to obtain the overall outlier score. It is recommended to use the ensemble method.',
```

Comment:
Not mandatory, but I think "normalise" is the UK spelling:
```suggestion
              'Sets the method that outlier detection uses. If not set, uses an ensemble of different methods, normalizes and combines their individual outlier scores to obtain the overall outlier score. It is recommended to use the ensemble method.',
```